ios/FluentUI/Vnext/Notification/MSFNotificationView.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// UIKit wrapper that exposes the SwiftUI `Notification` implementation
+@objc open class MSFNotification: NSObject, FluentUIWindowProvider {
+
+    /// The UIView representing the Notification.
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    /// Creates a new MSFNotification instance.
+    /// - Parameters:
+    ///   - style: The MSFNotification value used by the Notification.
+    ///   - message: The primary text to display in the Notification.
+    ///   - theme: The FluentUIStyle instance representing the theme to be overriden for this Notification.
+    @objc public init(style: MSFNotificationStyle,
+                      message: String,
+                      delayTime: TimeInterval,
+                      theme: FluentUIStyle? = nil) {
+        super.init()
+
+        notification = NotificationViewSwiftUI(style: style, message: message, delayTime: delayTime)
+        hostingController = FluentUIHostingController(rootView: AnyView(notification
+                                                                            .windowProvider(self)
+                                                                            .modifyIf(theme != nil, { notification in
+                                                                                notification.customTheme(theme!)
+                                                                            })))
+        hostingController.view.backgroundColor = UIColor.clear
+        hostingController.disableSafeAreaInsets()
+    }
+
+    @objc public var state: MSFNotificationState {
+        return notification.state
+    }
+
+    // MARK: - FluentUIWindowProvider
+
+    var window: UIWindow? {
+        return self.view.window
+    }
+
+    // MARK: - Show/Hide Methods
+
+    public func showNotification(in view: UIView, completion: ((MSFNotification) -> Void)? = nil) {
+        guard self.view.window == nil else {
+            return
+        }
+
+        let style = notification.tokens.style
+        let presentationOffset: CGFloat! = notification.tokens.presentationOffset
+        if style.isToast, let currentToast = MSFNotification.currentToast, currentToast.view.window != nil {
+            currentToast.hide {
+                self.showNotification(in: view, completion: completion)
+            }
+            return
+        }
+
+        hostingController.disableSafeAreaInsets()
+        var hostingControllerView: UIView! = hostingController.view
+        hostingControllerView = hostingController.view
+        hostingControllerView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
this will crash if hostingControllerView is nil.

you can validate it in the beginning of the func...  something like this:

```
guard let hostingControllerView = hostingController.view else {
    return
}
```